ocaml/idl/datamodel_host.ml

```diff
@@ -1744,6 +1744,34 @@ let set_https_only =
       ]
     ~allowed_roles:_R_POOL_OP ()
 
+let install_rpmgpgkey =
+  call ~name:"install_rpmgpgkey"
+    ~doc:"Install a RPM GPG public key to the host."
+    ~params:
+      [
+        (Ref _host, "self", "The host")
+      ; (String, "name", "The name of the RPM GPG public key")
+      ; (String, "pubkey", "The RPM GPG public key")
+      ; (String, "fingerprint", "The fingerprint of the RPM GPG public key")
+      ]
+    ~allowed_roles:(_R_POOL_OP ++ _R_CLIENT_CERT)
+    ~lifecycle:[] ()
+
+let uninstall_rpmgpgkey =
+  call ~name:"uninstall_rpmgpgkey"
+    ~doc:"Uninstall a RPM GPG public key on the pool."
```

Comment:
I think it should be "Uninstall an RPM" - but not a native speaker. While "r" is a consonant, "rpm" is pronounced like starting with a vowel and hence "an" would be correct.